fmriprep/workflows/bold/outputs.py

```diff
@@ -746,6 +746,14 @@ def init_ds_volumes_wf(
         ),
         name='inputnode',
     )
+    outputnode = pe.Node(
+        niu.IdentityInterface(
+            fields=[
+                'bold',
+            ],
+        ),
+        name='outputnode',
+    )
```

Comment:
I'm okay keeping this, even if we don't end up using it. `IdentityInterface` nodes get optimized away when building the execution graph. But this is a lot of lines:

```suggestion
    outputnode = pe.Node(niu.IdentityInterface(fields=['bold']), name='outputnode')
```

Reply:
I'm just going to remove it, since it won't be used in this PR and I want to minimize the diff.